### ISSUE TITLE
Bower install on windows vs linux differs

### ISSUE THREAD

**Issue by Deklin** [frapontillo/angular-gage#14]
Windows installs to:
![image](https://cloud.githubusercontent.com/assets/3372511/19556700/4c1ca15a-9690-11e6-809e-7cb7f23fb644.png)

Doing the same on linux puts al the files under a dist folder.

Any idea what is going on?

**Comment by frapontillo** [comment]
How did you run `dir`?

**Comment by Deklin** [comment]
@frapontillo I'm not sure I understand your statement?

Basically here is the issue we had locally in our build env
1. If we run a bower install on linux, it matches a bower install directly against your git repo for version 0.1.0, however if we run a bower install on windows, The download structure above is shown.

**Comment by Deklin** [comment]
Here is the result after a bower install on linux

![image](https://cloud.githubusercontent.com/assets/3372511/19557134/6294f142-9692-11e6-963c-83c22d37989d.png)

**Comment by frapontillo** [comment]
That _is_ weird, what version of bower are you using on Windows? Is it the same as it is on Linux?

**Comment by Deklin** [comment]
great question, checking

[EDIT] 1.7.9 on both

**Comment by Deklin** [comment]
I wonder if tagging a new one as 0.1.1 would fix it?

**Comment by frapontillo** [comment]
I don't think so, it's the same structure in 0.1.0. I am not on Windows, can you try reinstall bower?

**Comment by Deklin** [comment]
@frapontillo I've tried from multiple windows systems, same issue unfortunately. It is strange because we use a bunch of bower modules, this is the only one causing it. What I was hoping is that a 0.1.1 tag would bust any caches that may be  out there.

**Comment by frapontillo** [comment]
I have run it on Windows 10 and it works just fine:

![image](https://cloud.githubusercontent.com/assets/1140238/19562634/ecf89144-96d4-11e6-95a4-1ad8ffdc41f1.png)

**Comment by Deklin** [comment]
That is very odd, I wonder if something is mis-cached. I'll try another windows system off our corporate network.
